sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -104,21 +104,39 @@ private class LongSQLMetricParam(val stringValue: Seq[Long] => String, initialVa
   override def zero: LongSQLMetricValue = new LongSQLMetricValue(initialValue)
 }
 
+private[sql] object LongSQLMetricParam extends LongSQLMetricParam(_.sum.toString, 0L)
```

Comment:
Nit: change the object name `LongSQLMetricParam ` to something else? Compare to `StaticsLongSQLMetricParam`.